src/stylesheets/components/_header.scss

```diff
@@ -193,7 +193,7 @@ $z-index-nav:     9000;
       &[aria-expanded=false],
       &[aria-expanded=true] { /* stylelint-disable-line selector-no-qualifying-type */
         @include media($nav-width) { /* stylelint-disable-line selector-no-qualifying-type */
-          background-position: right 1.5rem top 50%;
+          background-position: right 1.5rem top 44%;
```

Comment:
Looking back at the issue I see where that was coming from now. However, we do have a new visual design lead so maybe @rtwell will care to chime in on how we should approach things going forward. 

Ryan, my proposed style is included above where the text is vertically centered, the dropdown starts below the nav, and the hover highlight is aligned with the bottom of the nav bar while remaining inside that container.